src/client.rs

```diff
@@ -836,8 +851,12 @@ mod tests {
             })
         ));
 
+        let key = match client.api_key.as_ref() {
+            Some(key) => key,
+            None => panic!("test_error_create_key no key to delete"),
+        };
```

Comment:
```suggestion
```

We can simply `.unwrap` the option here

Reply:
When I try to use a simple unrwrap here I get the following
```
cannot move out of an `Arc`
move occurs because value has type `std::option::Option<std::string::String>`, which does not implement the `Copy` traitrustc[E0507]
```
However if I implement it like 
```rust
client.api_key.as_deref().unwrap()
```
I don't get the error. Is that okay?